encrypted-media/content/drmconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "com.widevine.alpha": {
+        "drmtoday": true,
```

Comment:
@ddorwin I think was copied from the dash.js configuration JSON.  We need it in dash.js to help instantiate the correct JS object to interact with the DRMToday license server.  If we plan to support other license servers in the EME tests in the future, we would need a mechanism similar to this so that the test framework can use the correct JS functions to interact with that license server.  This is perhaps something we can add later, so we could remove this line.